pkg/kubernetes/client/v1alpha1/client.go

```diff
@@ -28,199 +28,252 @@ import (
 )
 
 const (
-	// K8sMasterIPEnvironmentKey is the environment variable key used to
-	// determine the kubernetes master IP address
+	// K8sMasterIPEnvironmentKey is the environment variable
+	// key used to provide kubernetes master IP address
 	K8sMasterIPEnvironmentKey env.ENVKey = "OPENEBS_IO_K8S_MASTER"
 
-	// KubeConfigEnvironmentKey is the environment variable key used to
-	// determine the kubernetes config
+	// KubeConfigEnvironmentKey is the environment variable
+	// key used to provide kubernetes config
 	KubeConfigEnvironmentKey env.ENVKey = "OPENEBS_IO_KUBE_CONFIG"
 )
 
-// getInClusterConfigFunc abstracts the logic to get
-// kubernetes incluster config
+// getInClusterConfigFn is a typed function
+// to abstract getting kubernetes incluster config
 //
 // NOTE:
 //  typed function makes it simple to mock
-type getInClusterConfigFunc func() (*rest.Config, error)
+type getInClusterConfigFn func() (*rest.Config, error)
 
-// buildConfigFromFlagsFunc provides the abstraction to get
-// kubernetes config from provided flags
+// buildConfigFromFlagsFn is a typed function
+// to abstract getting a kubernetes config from
+// provided flags
 //
 // NOTE:
 //  typed function makes it simple to mock
-type buildConfigFromFlagsFunc func(string, string) (*rest.Config, error)
+type buildConfigFromFlagsFn func(string, string) (*rest.Config, error)
```

Comment:
What is the idea behind this change ?  `Func` ----? `Fn`

Reply:
Func vs. Fn both seem appropriate to convey their purpose.
I chose `Fn` as it is shorter.
When the actual function names are large `Fn` when used as a suffix helps.